src/main/resources/messages.yml

```diff
@@ -0,0 +1,48 @@
+message:
+  exception:
+    fileNotFound: "Such file not found!"
+    cardFileNotReadable: "Can't read data from uploaded file!"
+    learningRegimeNotValid: "Value of Learning Regime is not valid - {0}!"
+    numbersOfCardsNegative: "Number of cards should be greater than 0!"
+    cardFieldsNotEmpty: "All of card fields should be filled!"
+    fileEmpty: "File is empty!"
+    fileWrongFormat: "Invalid format of file!"
+    fileDumpingFailed: "Dumping of file failed!"
+    fileCopyFailed: "Copy of file failed!"
+    deckAlreadyExists: "Such deck already exists!"
+    deckNotFound: "Such deck not found!"
+    fileSizeTooLarge: "File upload error: file is too large!"
+    imageFileWrongFormat: "File upload error: file is not an image!"
+    userCardQueueStatusNotValid: "Value of User Card Queue Status is not valid - {0}!"
+    numberOfPostponedDaysNegative: "Number of postponed days should be greater than 0!"
+    numberOfPostponedDaysLessThanPreviousLevel: >
+      Number of postponed days for this level should be greater than number of
+      postponed days for a previous one!
+    numberOfPostponedDaysMoreThanNextLevel: >
+      Number of postponed days for this level should be less than number of
+      postponed days for a next one!
+    emailNotExist: "Email doesn't exist!"
+    userNotFoundByEmail: "No user found with email {0}!"
+    userNotAuthorised: "Operation is unavailable for unauthorized users!"
+    ipAddressNotDetermined: "The IP address of a host could not be determined!"
+    mailNotSent: "Mail not sent!"
```

Comment:
gramar mistake
please rewrite messages without misakes

Reply:
Where did you see mistake?)

Reply:
line 7
write 
All of card's fields